kickstarter/android-oss
# NTV-466: Audio player

# 📲 What

Some project campaigns stories might have mp3 audios, the campaign tab now has the ability to reproduce those elements.


# 🛠 How
- Using `MediaPlayer` with `prepareAsync` and built in controllers for the audio to play.
- The ViewHolder that contains the audioPlayer is aware of the parent fragment events

# 👀 See

|  |  |

# 📋 QA
- Load this project https://staging.kickstarter.com/projects/gkdreamcatcher/george-kahn-jazz-and-blues-revue-our-first-studio it contains two audios on the campaign
- Make sure you can navigate between tabs
- Make sure you can go to background
- Make sure you can scroll properly the story
- Make sure you can reproduce the audio

# Story 📖

[NTV-466](https://kickstarter.atlassian.net/browse/NTV-466)
[NTV-458](https://kickstarter.atlassian.net/browse/NTV-458)
